cohort-evaluator-spark/src/test/java/com/ibm/cohort/cql/spark/data/SparkDataRowTest.java

```diff
@@ -153,8 +153,8 @@ public void testConversionSemanticsNullValues() {
             assertEquals( pojo.getShortField(), sdr.getValue("shortField") );
             assertEquals( pojo.getIntegerField(), sdr.getValue("integerField") );
             assertEquals( pojo.getLongField(), sdr.getValue("longField") );
-            assertEquals( pojo.getFloatField(), (BigDecimal) sdr.getValue("floatField") );
```

Comment:
Comparing incompatible types should fail (it did not because this is the null test).  

I recognize that in other tests we're converting the float and double fields to BigDecimals, so if the preference is to cast pojo.getFloatField() to a BigDecimal instead of removing it from the right side I am willing to change.